.github/workflows/tests.yml

```
@@ -70,6 +70,28 @@ jobs:
           name: bazel-testlogs
           path: bazel-testlogs
 
+  lts_ios_integration_tests:
+    name: Build and Test ( Virtual Frameworks + LTS )
+    runs-on: macos-12
+    steps:
+      - uses: actions/checkout@v3
+      - name: Select Xcode
+        run: .github/workflows/xcode_select.sh
+      - name: Build and Test
+        run: |
+          export USE_BAZEL_VERSION=5.3.2
+          bazelisk build --features apple.virtualize_frameworks \
+              --local_test_jobs=1 \
+              --apple_platform_type=ios \
+              --deleted_packages='' \
```

Since this is all configured in `xcode_select.sh`, could simplify this a bit:
```suggestion
              --config=ios \
```